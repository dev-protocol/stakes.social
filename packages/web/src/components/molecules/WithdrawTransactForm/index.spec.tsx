import React from 'react'
import Web3 from 'web3'
import { render } from '@testing-library/react'
import { WithdrawTransactForm } from '.'
import 'src/__mocks__/window/matchMedia.mock'
import WalletContext from 'src/context/walletContext'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/uniswap/hooks')
jest.mock('src/fixtures/wallet/hooks')
jest.mock('web3')
jest.mock('@dev/graphql')

describe(`${WithdrawTransactForm.name}`, () => {
  test('Snapshot', () => {
    const component = render(
      <WalletContext.Provider value={{ web3: new Web3(), setProviders: () => {} }}>
        <WithdrawTransactForm onClickMax={() => {}} propertyAddress="" withdraw={() => {}} />
      </WalletContext.Provider>
    )
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
