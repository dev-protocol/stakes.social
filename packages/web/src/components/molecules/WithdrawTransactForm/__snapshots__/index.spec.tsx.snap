// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithdrawTransactForm Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-hKwDye gWsDYA"
      style="opacity: 1;"
    >
      <div
        class="ant-radio-group ant-radio-group-outline"
        style="margin-bottom: 12px;"
      >
        <div
          class="ant-space ant-space-vertical"
        >
          <div
            class="ant-space-item"
            style="margin-bottom: 8px;"
          >
            <label
              class="ant-radio-wrapper"
            >
              <span
                class="ant-radio"
              >
                <input
                  class="ant-radio-input"
                  type="radio"
                  value="2"
                />
                <span
                  class="ant-radio-inner"
                />
              </span>
              <span>
                <div
                  class="ant-space ant-space-horizontal ant-space-align-center"
                >
                  <div
                    class="ant-space-item"
                    style="margin-right: 24px;"
                  >
                    <div
                      style="margin-left: 8px; min-width: 40px;"
                    >
                      #
                      2
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                    style="margin-right: 24px;"
                  >
                    <div
                      style="min-width: 120px;"
                    >
                      <span>
                        1e-15
                      </span>
                      <span
                        style="font-size: 0.6em; margin-left: 2px;"
                      >
                        Staked
                      </span>
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                  >
                    <div>
                      <span>
                        0
                      </span>
                      <span
                        style="font-size: 0.6em; margin-left: 2px;"
                      >
                        Claimable
                      </span>
                    </div>
                  </div>
                </div>
              </span>
            </label>
          </div>
          <div
            class="ant-space-item"
            style="margin-bottom: 8px;"
          >
            <label
              class="ant-radio-wrapper"
            >
              <span
                class="ant-radio"
              >
                <input
                  class="ant-radio-input"
                  type="radio"
                  value="4"
                />
                <span
                  class="ant-radio-inner"
                />
              </span>
              <span>
                <div
                  class="ant-space ant-space-horizontal ant-space-align-center"
                >
                  <div
                    class="ant-space-item"
                    style="margin-right: 24px;"
                  >
                    <div
                      style="margin-left: 8px; min-width: 40px;"
                    >
                      #
                      4
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                    style="margin-right: 24px;"
                  >
                    <div
                      style="min-width: 120px;"
                    >
                      <span>
                        1e-15
                      </span>
                      <span
                        style="font-size: 0.6em; margin-left: 2px;"
                      >
                        Staked
                      </span>
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                  >
                    <div>
                      <span>
                        0
                      </span>
                      <span
                        style="font-size: 0.6em; margin-left: 2px;"
                      >
                        Claimable
                      </span>
                    </div>
                  </div>
                </div>
              </span>
            </label>
          </div>
          <div
            class="ant-space-item"
          >
            <label
              class="ant-radio-wrapper"
            >
              <span
                class="ant-radio"
              >
                <input
                  class="ant-radio-input"
                  type="radio"
                  value="6"
                />
                <span
                  class="ant-radio-inner"
                />
              </span>
              <span>
                <div
                  class="ant-space ant-space-horizontal ant-space-align-center"
                >
                  <div
                    class="ant-space-item"
                    style="margin-right: 24px;"
                  >
                    <div
                      style="margin-left: 8px; min-width: 40px;"
                    >
                      #
                      6
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                    style="margin-right: 24px;"
                  >
                    <div
                      style="min-width: 120px;"
                    >
                      <span>
                        1e-15
                      </span>
                      <span
                        style="font-size: 0.6em; margin-left: 2px;"
                      >
                        Staked
                      </span>
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                  >
                    <div>
                      <span>
                        0
                      </span>
                      <span
                        style="font-size: 0.6em; margin-left: 2px;"
                      >
                        Claimable
                      </span>
                    </div>
                  </div>
                </div>
              </span>
            </label>
          </div>
        </div>
      </div>
      <button
        class="ant-btn ant-btn-default ant-btn-lg sc-gsDKAQ sc-eCImPb ldRQYr cVmGpy"
        type="button"
      />
    </div>
  </div>
</body>
`;
